packages/details/index.tsx

```diff
@@ -57,25 +54,23 @@ const mapStateToProps = (state: IRootState) => {
 };
 
 const mapDispatchToProps = (dispatch) => ({
```

Comment:
I wonder if it would be possible to use `bindActionCreators` here.

Reply:
Yes, it was possible, I've just changed it to `bindActionCreators` as you suggested.